qemu/tests/virtio_serial_throttling.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from virttest import error_context, env_process
+from virttest.utils_virtio_port import VirtioPortTest
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Induce throttling by opening chardev but not reading data from it:
+    1) Start guest with virtio-serial with unix option
+    2) On the host, open the chardev but don't read from it
+    3) In the guest, write to the virtio-serial port.
+    4) transfer data from host to guest but do not read in guest
+    5) repeat step 1 to 4 with tcp option.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    @error_context.context_aware
+    def send_data_from_guest_to_host():
+        session = vm.wait_for_login()
+        port.open()
+        error_context.context("send data from guest to host", logging.info)
+        if params['os_type'] == 'windows':
+            vport_name = '\\\\.\\Global\\' + port.name
+            cmd = 'dd if=/dev/zero of=%s bs=1024 count=1' % vport_name
```

Comment:
@michiboo This is only available when test with internal winutils.iso, otherwise, we can not use dd command for windows  guest.  We do not forbid to use it, but i prefer to use `windows_support.WinBufferedReadFile`. It is up to you.

Reply:
I will use DD since it just come with window

Reply:
Ack, ignore my comment above, it is not correct.